### PR TITLE
Added content about Drupal SEO

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**

### CODE REVIEW

File: content/docs/how-we-work/drupal/drupal-seo.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Drupal SEO
+weight: 7
+---
+
+# Drupal SEO
+
+We configure sites that are optimized for SEO”, which improves their visibility on search engines like Google.
```
Comment (hussainweb) [inline content/docs/how-we-work/drupal/drupal-seo.md:8]: There is a closing double-quote after SEO. It seems like a typo.

File: content/docs/how-we-work/drupal/drupal-seo.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+One of the key aspects of SEO is the use of specific modules that can enhance a site’s search engine performance. For instance, the [XML Sitemap](https://www.drupal.org/project/xmlsitemap) module creates a map of your site that helps search engines understand its structure and efficiently crawl through its pages. The [Google Analytics](https://www.drupal.org/project/google_analytics) module integrates our site with Google’s analytics tool, providing valuable insights into how visitors interact with our site. The [Metatag](https://www.drupal.org/project/metatag) module allows us to add meta tags to our site, which are snippets of text that describe a page’s content and can influence how your website is understood and displayed by search engines.
```
Comment (hussainweb) [inline content/docs/how-we-work/drupal/drupal-seo.md:10]: Yoast is another one we can mention. But please check if it still in development for Drupal.

File: content/docs/how-we-work/drupal/drupal-seo.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Another useful module is [Pathauto](https://www.drupal.org/project/pathauto), which automatically generates URL aliases for various types of content on our site. Proper configuration of this module is crucial because URLs play an important role in SEO. The [Redirect](https://www.drupal.org/project/redirect) module is also essential as it allows you to set up URL redirects, preventing “page not found” errors when pages are moved or renamed.
+
+There are SEO practices we follow. We provide training to our clients on how to create content that is optimized for search engines. This includes using keywords effectively, writing clear and compelling meta descriptions, using headings correctly, and more. We also conduct keyword research during the discovery phase of the project. Keywords are the words and phrases that people type into search engines, and by understanding what keywords are relevant to our clients’ businesses, we can help them create content that will attract the right kind of visitors.
```
Comment (hussainweb) [inline content/docs/how-we-work/drupal/drupal-seo.md:14]: Can you confirm if we actually provide training? I can’t recall if we have ever done that. If not, we can just remove that phrase and the rest of the content is fine (will just need to be rewritten to flow in a sentence).

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: content/docs/how-we-work/drupal/drupal-seo.md (hunk 2 above)
Comment (axelabhay) [inline content/docs/how-we-work/drupal/drupal-seo.md:10]: @Yemaneberhan-Lemma Just thought to check when we say `One of the key aspects of SEO is the use of specific modules...`, is using specific modules are the aspects of SEO? To me it sounds like using modules are the features of SEO.

**Comment by axelabhay** [comment]
@Yemaneberhan-Lemma, @hussainweb, can SEO introduction participate in [Common Areas of concern with web projects](https://axelerant.github.io/engg-handbook/docs/common-areas/) page?

**Review by hussainweb — APPROVED** [review]

**Pull request merged**